utils/Socket.js

```diff
@@ -47,13 +46,29 @@ function sendDeviceCommand(eventName, payload, cb, options) {
               break;
             case 'register-ok':
               debug('Register ok');
-              var theEvent = {
-                t: eventName,
-                deviceId: admatrix.config.device.identifier,
-                p: payload
-              };
-              debug('[ss]', theEvent);
-              // don't like this
+              var theEvent = {};
+              //groups case
+              if(_.isArray(admatrix.config.device.identifier)) {               
+                async.map(admatrix.config.device.identifier, function (did, callback) {
```

Comment:
return this because it needs to only fire once, otherwise it will send an empty event

Reply:
line 72 will still fire